catalyst-gateway/bin/src/service/common/objects/fragment_status.rs

```diff
@@ -0,0 +1,57 @@
+//! Defines API schemas of fragment status types.
+
+use poem_openapi::{types::Example, Object, Union};
+
+use crate::service::common::objects::{block::BlockDate, hash::Hash};
+
+#[derive(Object)]
+#[oai(example = false)]
```

Comment:
If possible, the OpenAPI structure types in this fle should also be marked `deprecated`.

Reply:
I didn't find anything allowing to mark schemas as deprecated. I guess we can add it to the schema's description, so I did that. https://github.com/input-output-hk/catalyst-voices/pull/93/files#diff-fb3f50253111b6caedbb2fe1053720ba24c224ce3b9c7f92e48a10a16660447aR9